docs/06-concepts/11-authentication/02-basics.md

```diff
@@ -96,3 +96,31 @@ class MyEndpoint extends Endpoint {
 :::caution
 Keep in mind that a scope is merely an arbitrary string and can be written in any format you prefer. However, it's crucial to use unique strings for each scope, as duplicated scope strings may lead to unintentional data exposure.
 :::
+
+## Managing Authentication in Serverpod's `StatusEndpoint`
+
+The `StatusEndpoint` class includes methods for handling user sign-outs, whether from a single device, all devices, or via a deprecated method.
+
+- **Sign out from the current device**
+
+    Use the `signOutDevice` method to log the user out of the current session only, leaving other sessions active.
+
+    ```dart
+    await endpoints.status.signOutDevice(session);
+    ```
+
+- **Sign out from all devices**
+
+    Use the `signOutAllDevices` method to log the user out from all devices, effectively ending all active sessions.
+
+    ```dart
+    await endpoints.status.signOutAllDevices(session);
+    ```
+
+- **Deprecated method for signing out from all devices**
+
+    The `signOut` method is deprecated and will be removed in a future release. It currently allows signing out from all devices, but new implementations should use `signOutDevice` or `signOutAllDevices`.
```

Comment:
Suggestion: Don't mention it will be removed since that would break compatability with older clients. Only mention that it it deprecated and the other methods should be used instead.

Reply:
Suggestion: Add a description that the behavior of the method is determined based on the configuration option, and link to that option.